src/SwedbankPay.Sdk/PaymentInstruments/Trustly/TrustlyPaymentData.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Trustly
+{
+    public class TrustlyPaymentData
```

Comment:
How about naming the `Payment` classes `Detail` like this, and using `Options` for the payment instrument specific stuff?

```suggestion
    public class TrustlyPaymentDetails
```